test/integration/xfcc_integration_test.cc

```diff
@@ -484,7 +484,7 @@ TEST_P(XfccIntegrationTest, TagExtractedNameGenerationTest) {
       {listenerStatPrefix("ssl.ocsp_staple_failed"), "listener.ssl.ocsp_staple_failed"},
       {listenerStatPrefix("http.config_test.downstream_rq_2xx"), "listener.http.downstream_rq_xx"},
       {listenerStatPrefix("http.config_test.downstream_rq_5xx"), "listener.http.downstream_rq_xx"},
-      {listenerStatPrefix("worker_0.downstream_cx_total"), "listener.worker_0.downstream_cx_total"},
+      {listenerStatPrefix("worker_0.downstream_cx_total"), "listener.downstream_cx_total"},
```

Comment:
Should we consider backing this (and the other similar/related) changes out before the release, and figuring out a way to "version" the stats names in the next release? cc @davinci26

Reply:
I am sorry to hear that I caused you an issue/frustration 

> for future reference: these kinds of changes can be breaking. This one was for us.

There is some discussion on the backwards compatibility aspect of this change in 
https://github.com/envoyproxy/envoy/issues/20201#issuecomment-1151600644

> Should we consider backing this (and the other similar/related) changes out before the release, and figuring out a way to "version" the stats names in the next release?

Can you handle backwards compatibility in PromQL? My mild preference would be to fix the clients since the change is moving the stats in the correct direction but I am ok with any direction.

If you want we can revert and iterate on the approach mentioned in the issue above.

I leave it up to you. I can revert tomorrow morning if this is the approach we want to take.

Reply:
@pradeepcrao @yanavlasov 

Yes I don't object to the direction this is taking, and we might be able to fix forward. Pradeep is looking at this.

In the future maybe we can do some more reach-out and warning on potentially breaking changes?

Reply:
Also I was tagged on the referenced PR so it was my bad for not commenting :)